.github/workflows/frontend-check-core-changes.yml

```
@@ -0,0 +1,38 @@
+name: Comment on Core Files changed
+on:
+  workflow_call:
+    secrets:
+      github-token:
+        required: true
+
+jobs:
+  core-files-modified:
+    name: Core Files Modified
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check changed files
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.github-token }}
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Get modified core files excluding the app files
+        id: modified-files-excluded
+        uses: tj-actions/changed-files@v35
+        with:
+          files_ignore_separator: ","
+          dir_names: true
+          files: |
+            ClientApp/src/**/*.{ts,html,scss}
+          files_ignore: | 
+            *.app.*, index.*
```

```suggestion
          files: "ClientApp/src/**/*.{ts,html,scss}"
          files_ignore: "*.app.*, index.*"
```
Since it's not multiline no need for "|"